data/resources/ui/content-message-document.ui

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <template class="ContentMessageDocument" parent="ContentMessageBase">
+    <property name="layout-manager">
+      <object class="GtkBoxLayout">
+        <property name="orientation">vertical</property>
+      </object>
+    </property>
+    <style>
+      <class name="message-bubble" />
+    </style>
+    <child>
+      <object class="GtkLabel" id="sender_label">
+        <property name="ellipsize">end</property>
+        <property name="single-line-mode">True</property>
+        <property name="xalign">0</property>
+        <style>
+          <class name="sender-text" />
+        </style>
+      </object>
+    </child>
+    <child>
+      <object class="AdwActionRow" id="file_row">
```

Comment:
I would use a regular `GtkBox` here since this widget is designed for a different use case and it's more difficult to style as we want. This is roughly the layout we need:

```
GtkBox::horizontal
 ______________________
|           |          |
|           | GtkLabel |
| GtkButton |----------|
|           | GtkLabel |
|___________|__________|
             GtkBox::vertical
```